build-system/tasks/bundle-size.js

```diff
@@ -86,8 +81,6 @@ function getBrotliBundleSize() {
 /**
  * Store the bundle size of a commit hash in the build artifacts storage
  * repository to the passed value.
- *
- * @return {!Promise}
```

Comment:
an async function always returns a promise

Reply:
But do we have to mark it as such? Or is it inferred from the `async` keyword?

Reply:
Seems like closure compiler infers async functions return `!<Promise<?>` so you're ok there. But might be good to keep consistent with the rest of the repo?

Reply:
I've been writing the actual return value type for async functions without wrapping them in a promise, assuming the `async` bit implied the promise. I'm not sure what the recommended practice is.